src/types.ts

```diff
@@ -46,6 +48,13 @@ export interface Weather extends HassEntity {
   };
 }
 
+export interface TemperatureSensor extends HassEntity {
+  state: string;
+  attributes: {
+    unit_of_measurement: TemperatureUnit;
```

Comment:
`unit_of_measurement` is not used, or?

Reply:
used it in getLocalizedTemperature() now